### DISCUSSION TITLE
Upgrading k3s from 1.22.4 to 1.22.7 without enabling CCM

### DISCUSSION THREAD

**Discussion by ssharma2089** [k3s-io/k3s#6491] [General]
We want to upgrade k3s from 1.22.4 to 1.22.7. We have disabled CCM. 
What is the suggested way  to upgrade if we have been using service loadbalancer but not deploying CCM.

**Comment by brandond** [comment]
You can continue to operate it just as before. As described at https://github.com/k3s-io/k3s/blob/master/docs/adrs/servicelb-ccm.md, disabling the CCM still disables the same functionality as before. If you run into problems with port conflicts due to the embedded CCM's metrics listener, you can disable or reassign it using existing flags.
